File: CILantroTools/cilantro-tools-web-ui/src/api/models/tests/input-output/InputOutputLine.ts
import AbstractInputOutputElement from './elements/AbstractInputOutputElement';
import InputOutput from './InputOutput';

interface InputOutputLine {
	elements: AbstractInputOutputElement[];

	isRepeatBlock: boolean;
	repeatBlockInputOutput?: InputOutput;
	repeatBlockMin?: string;
	repeatBlockMax?: string;
}

export default InputOutputLine;

File: CILantroTools/CILantroToolsWebAPI/CILantroToolsWebAPI/Models/Tests/InputOutput/InputOutputLine.cs
using CILantroToolsWebAPI.Models.Tests.InputOutput.Elements;

namespace CILantroToolsWebAPI.Models.Tests.InputOutput
{
    public class InputOutputLine
    {
        public AbstractInputOutputElement[] Elements { get; set; } = new AbstractInputOutputElement[0];



        public bool IsRepeatBlock { get; set; }

        public InputOutput RepeatBlockInputOutput { get; set; }

        public string RepeatBlockMin { get; set; }

        public string RepeatBlockMax { get; set; }
    }
}